src/data/mockDSData.js

```diff
@@ -0,0 +1,9 @@
+let ethnicity=[
+{id:"black",value:"black",label:"Black"},
+{id:"mixed",value:"mixed",label:"Mixed"},
+{id:"white",value:"white",label:"White"},
+{id:"other",value:"other",label:"Other"},
+{id:"asian",value:"asian",label:"Asian"},
+]
```

Comment:
small thing - could you put this in a folder `src/data/test/..`
if we start doing mock data files we want to make sure they don't overwhelm the rest of the code

Reply:
I've moved the mock data file into a new test directory under the data directory `src/data/test/`.